### ISSUE TITLE
endpoint_status not prefetched in findings_list

### ISSUE THREAD

**Issue by valentijnscholten** [DefectDojo/django-DefectDojo#3234]
search is slow again:
![image](https://user-images.githubusercontent.com/4426050/98715769-df13ae00-238a-11eb-96f0-72ce9cf97359.png)

**Comment by valentijnscholten** [comment]
it's this code. if you have a bunch of findings and endpoints you can end up with 600+ queries to render a page.
```
@register.filter
def endpoint_update_time(endpoint, finding):
    status = Endpoint_Status.objects.get(endpoint=endpoint, finding=finding)
    return status.last_modified


@register.filter
def endpoint_date(endpoint, finding):
    status = Endpoint_Status.objects.get(endpoint=endpoint, finding=finding)
    return status.date


@register.filter
def endpoint_mitigator(endpoint, finding):
    status = Endpoint_Status.objects.get(endpoint=endpoint, finding=finding)
    return status.mitigated_by


@register.filter
def endpoint_mitigated_time(endpoint, finding):
    status = Endpoint_Status.objects.get(endpoint=endpoint, finding=finding)
    return status.mitigated_time


@register.filter
def endpoint_check_active(endpoint, finding):
    status = Endpoint_Status.objects.get(endpoint=endpoint, finding=finding)
    return not status.mitigated
```

**Comment by valentijnscholten** [comment]
fixes in dev, so 1.10.0
